experimental/ExteriorAlgebra/ExteriorAlgebra_naive.jl

```diff
@@ -0,0 +1,95 @@
+export exterior_algebra_naive;  # constructor
+export ContainsDuplicates;   # export just for developing/testing
+
+# Auxiliary function -- worth keeping?
+
+function ContainsDuplicates(L)  # L must be indexable (1:n), and its elems must be sortable
+  n = length(L);
+  if (n < 2)  return false;  end;
```

Comment:
Could also be 
```
n < 2 && return false
```

Reply:
Thanks!  It is a question of style.  I prefer to write explicitly `if... end` because I find it easier to read.  I presume that compilation will produce the same code in both cases.